shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/impl/EncryptAlterTableTokenGenerator.java

```diff
@@ -53,7 +53,7 @@ protected boolean isGenerateSQLTokenForEncrypt(final SQLStatementContext sqlStat
         result.addAll(getDropColumnTokens(tableName, alterTableStatementContext.getSqlStatement().getDropColumnDefinitions()));
         return result;
     }
-    
```

Comment:
@arthasking123 Keep indents consistent with the previous one. Please modify similar issues in other places.